ChangeLog.d/changes.txt

```diff
@@ -0,0 +1,3 @@
+Changes
```

Comment:
Please use a filename unique to this change. `changes.txt` is too generic.

Reply:
The name was changed.

However, the guidelines mention only this: `A changelog entry file must have the extension `*.txt``.
It might be assumed that all this is automatic, and any valid name will do (as it did previously).
Maybe the instructions in `00README.md` should be updated.

Reply:
`changes.txt` probably would have been OK but there's a risk that someone else might have chosen that name for another PR and caused a conflict. I agree the instructions probably should be updated - we can look into that (unless you want to submit a separate PR?)